apps/microsoft-teams/frontend/src/components/config/ChannelSelectionModal/ChannelSelectionModal.tsx

```diff
@@ -1,100 +1,100 @@
 import { useEffect, useState } from 'react';
-import {
-  Button,
-  FormControl,
-  Modal,
-  Paragraph,
-  Radio,
-  Table,
-  TextLink,
-} from '@contentful/f36-components';
+import { FormControl, Modal, Radio, Table } from '@contentful/f36-components';
 import { appDeepLink, channelSelection } from '@constants/configCopy';
 import { styles } from './ChannelSelectionModal.styles';
 import { Notification, TeamsChannel } from '@customTypes/configPage';
 import ModalHeader from '@components/config/ModalHeader/ModalHeader';
 import TeamsLogo from '@components/config/TeamsLogo/TeamsLogo';
 import EmptyState from '@components/config/EmptyState/EmptyState';
 import EmptyFishbowl from '@components/config/EmptyState/EmptyFishbowl';
+import ChannelsErrorMessage from '@components/config/ChannelsErrorMessage/ChannelsErrorMessage';
 import { defaultNotification } from '@constants/defaultParams';
+import ChannelSelectionSupplementalModalContent from '../ChannelSelectionSupplementalModalContent/ChannelSelectionSupplementalModalContent';
 
 interface ChannelSelectionModalProps {
   isShown: boolean;
   onClose: () => void;
   savedChannel: TeamsChannel;
   handleNotificationEdit: (notificationEdit: Partial<Notification>) => void;
   channels: TeamsChannel[];
+  error: boolean;
 }
 
 const ChannelSelectionModal = (props: ChannelSelectionModalProps) => {
-  const { isShown, onClose, savedChannel, handleNotificationEdit, channels } = props;
+  const { isShown, onClose, savedChannel, handleNotificationEdit, channels, error } = props;
   const [selectedChannel, setSelectedChannel] = useState<TeamsChannel>(
     savedChannel ?? defaultNotification.channel
   );
-  const { title, button, link, emptyContent, emptyHeading, description } = channelSelection.modal;
+  const { title, link, emptyContent, emptyHeading, errorMessage } = channelSelection.modal;
 
   useEffect(() => {
     const foundChannel = channels.find((channel) => channel.id === savedChannel.id);
     setSelectedChannel(foundChannel ?? defaultNotification.channel);
   }, [savedChannel]);
 
+  const SupplementalModalContent = ({ children }: { children: React.ReactNode }) => (
+    <ChannelSelectionSupplementalModalContent
+      onClose={onClose}
+      handleNotificationEdit={handleNotificationEdit}
+      selectedChannel={selectedChannel}>
+      {children}
+    </ChannelSelectionSupplementalModalContent>
+  );
```

Comment:
I originally abstracted this logic given the designed loading state in the modal shared these components as well. But since I have removed those components, opting for the dot loading within the button, I left this anyway to simplify the JSX in this file, but am totally fine reverting!!